x-pack/plugins/session_view/public/components/process_tree_alert/index.tsx

```diff
@@ -76,12 +95,42 @@ export const ProcessTreeAlert = ({
           />
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
-          <EuiIcon type="alert" color="danger" />
+          <EuiToolTip
+            position="top"
+            content={`${capitalize(processEventAlertCategory)} ${
+              capitalize(processEventAlertCategory) !== ALERT ? ALERT : ''
```

Comment:
wouldn't 'alert' always not equal 'Alert' because of the capitalize?

Reply:
maybe adding a getAlertTypeTooltip function would be cleaner.

Reply:
also, we'll have to localize this as well, since it's user facing text. so better to have 3 localized strings for each type and just have  switch statement to return the one you need to show.